src/main/java/scrolls/elder/model/Model.java

```diff
@@ -89,4 +89,18 @@ public interface Model {
      * @throws NullPointerException if {@code predicate} is null.
      */
     void updateFilteredPersonList(Predicate<Person> predicate);
+
+    /**
+     * Updates the filter of the filtered volunteer list to filter by the given {@code predicate}.
+     * Befriendee list is not filtered.
+     * @throws NullPointerException if {@code predicate} is null.
+     */
+    void updateFilteredVolunteerList(Predicate<Person> predicate);
+
+    /**
+     * Updates the filter of the filtered befriendee list to filter by the given {@code predicate}.
+     * Volunteer list is not filtered.
+     * @throws NullPointerException if {@code predicate} is null.
+     */
+    void updateFilteredBefriendeeList(Predicate<Person> predicate);
```

Comment:
Not sure if these commands have been phased out through jia en's latest pr, may have to refactor this.